libcxx/include/__format/format_arg.h

```diff
@@ -267,7 +366,8 @@ class _LIBCPP_TEMPLATE_VIS basic_format_arg<_Context>::handle {
 // This function is user facing, so it must wrap the non-standard types of
 // the "variant" in a handle to stay conforming. See __arg_t for more details.
 template <class _Visitor, class _Context>
-_LIBCPP_HIDE_FROM_ABI decltype(auto) visit_format_arg(_Visitor&& __vis, basic_format_arg<_Context> __arg) {
+_LIBCPP_DEPRECATED_IN_CXX26 _LIBCPP_HIDE_FROM_ABI decltype(auto)
```

Comment:
let's not show a deprecated message when the replacement can't be used by the user's compiler.
```suggestion
#ifdef __cpp_explicit_this_parameter
_LIBCPP_DEPRECATED_IN_CXX26 
#endif
_LIBCPP_HIDE_FROM_ABI decltype(auto)
```